lib/Scheduler/aco.cpp

```diff
@@ -61,6 +62,13 @@ ACOScheduler::ACOScheduler(DataDepGraph *dataDepGraph,
   */
   int pheremone_size = (count_ + 1) * count_;
   pheremone_ = new pheremone_t[pheremone_size];
+
+  this->initialSchedule_ = NULL;
+  if (initialSchedule)
```

Comment:
Try to follow LLVM coding style if you can. We are trying to stick to that standard for future patches.